haproxy_exporter.go

```diff
@@ -437,13 +437,22 @@ func filterServerMetrics(filter string) (map[int]*prometheus.GaugeVec, error) {
 }
 
 func main() {
+	const pidFileHelpText = `Path to HAProxy pid file.
```

Comment:
I think best practise in Go is to have constants in package scope rather than bloating functions. What do you think?